openmdao/docs/basic_guide/first_analysis.rst

```diff
@@ -1,12 +1,12 @@
 .. _tutorial_paraboloid_analysis:
 
-In the last tutorial, we discussed the three basic kinds of components in the OpenMDAO framework.
-This tutorial focuses on using :ref:`ExplicitComponent <comp-type-2-explicitcomp>` to build a simple analysis of a paraboloid function.
-We'll explain the basic structure of a run file and show you how to set inputs, run the model, and check the output files
+In the previous tutorial, we discussed the three basic kinds of Components in the OpenMDAO framework.
+This tutorial focuses on using one of those, :ref:`ExplicitComponent <comp-type-2-explicitcomp>`, to build a simple analysis of a paraboloid function.
+We'll explain the basic structure of a run file, show you how to set inputs, run the model, and check the output files.
 
-******************************************
-Paraboloid - A Single Disciplinary Model
-******************************************
+*************************************
+Paraboloid - A Monodisciplinary Model
```

Comment:
"Single Discipline" 

No one says "monodisciplinary" ...

Reply:
Dang, too bad, because it's actually a real word! 😄